modules/ossm-rn-new-features.adoc

```diff
@@ -43,6 +43,10 @@ This release of {ProductName} adds support for a disconnected installation.
 
 This release of {ProductName} adds support for Istio 1.4.6 and Jaeger 1.17.1.
```

Comment:
The entry below is fine but perhaps we could update this line here because we omitted Kiali in this note:

This release of {ProductName} adds support for Istio 1.4.6, Kiali 1.12.7 and Jaeger 1.17.1.

Reply:
@jshaughn Was that release of Kiali added in 1.1.0 release or is that the current release of Kiali?

Reply:
@neal-timpe Kiali 1.12.7 is the Kiali release included in SM 1.1.0.  It is not the most recent version, but is much newer than the version in SM 1.0 (which is Kiali 1.0)

Reply:
Ok, done.